docs/search relevance/test_2.md

```diff
@@ -52,4 +52,4 @@ We'd like to test this on the live site with people outside Wellcome, possibly c
 
 ## Results
 
-## Conclusions
+minimum should match + boost did best! Default query changed accordingly
```

Comment:
Graphs / data / tables interpretive dance proving this?

Reply:
I've been looking, but I can't find any of the numbers / graphs from the time! 

If i'm remembering correctly, we used the CTR, CTR5, and CTR10 metrics in the reporting dashboard to make the decision about which was performing best. Since the switch to the new index, metrics, and dashboards, a few things have broken and I'm not sure it's worth recreating them for a decision that's now out of date.

Reply:
Agreed, it was the most boring test anyway.